cmd/config.go

```diff
@@ -361,7 +371,12 @@ func (c *GRPCClientConfig) MakeTargetAndHostOverride() (string, string, error) {
 		}
 		// Lookup backends using DNS SRV records.
 		targetHost := c.SRVLookup.Service + "." + c.SRVLookup.Domain
-		return fmt.Sprintf("srv://%s/%s", c.DNSAuthority, targetHost), targetHost, nil
+
+		hostOverride = targetHost
+		if c.HostOverride != "" {
+			hostOverride = c.HostOverride
+		}
+		return fmt.Sprintf("srv://%s/%s", c.DNSAuthority, targetHost), hostOverride, nil
 
 	} else {
```

Comment:
unrelated, can be addressed in some other PR -- I think this final block would be better structured as
```go
} else if c.ServerIPAddresses != nil {
  return "static:///"...
} else {
  return "", "", errors.New(...)
}
```